src/lib_ccx/ccx_common_option.h

```diff
@@ -87,7 +87,7 @@ struct encoder_cfg
 struct ccx_s_options // Options from user parameters
 {
 	int extract;                                               // Extract 1st, 2nd or both fields
-	int no_rollup;
+	int no_rollup;											   // If 1, write one line at a time
```

Comment:
This disables roll-up emulation (which causes duplicate output when you open the resulting file with a text editor, but is correct if you actually _play_ it with a media player)

Reply:
Thanks! Such things must to be commented! For me (a non-English speaker with a very little knowledge of cc) no-rollup option was a mystery.